packages/@aws-cdk/aws-efs/lib/access-point.ts

```diff
@@ -109,23 +114,62 @@ export interface AccessPointProps extends AccessPointOptions {
   readonly fileSystem: IFileSystem;
 }
 
+/**
+ * Attributes that can be specified when importing an AccessPoint
+ */
+export interface AccessPointAttributes {
+  /**
+   * The ID of the AccessPoint
+   * One of this, of {@link accessPointArn} is required
+   *
+   * @default - no access point id
```

Comment:
Not true. It should be

```suggestion
   * @default - determined based on accessPointArn
```